draft-schinazi-connect-udp-listen.md

```diff
@@ -145,12 +142,12 @@ octets" in {{UDP}}).
 ## The connect-udp-listen Header Field {#hdr}
 
 The "connect-udp-listen" header field is an Item Structured Field, see {{Section
-3.3 of !STRUCT-FIELD=RFC8941}}; its value MUST be an Integer; any other value
-type MUST be handled as if the field were not present by recipients (for
-example, if this field is included multiple times, its type will become a List
-and the field will therefore be ignored). This document does not define any
+3.3 of !STRUCT-FIELD=RFC8941}}; its value MUST be an Integer and set as the Context ID. Any other value
+type MUST be handled as if the field were not present by the recipients (for
+example, if this field is defined multiple times, its type becomes a List
+and therefore is to be ignored). This document does not define other
 parameters for the connect-udp-listen header field value, but future documents
-might define parameters. Receivers MUST ignore unknown parameters.
+might do the same. Receivers MUST ignore unknown parameters.
```

Comment:
This is backwards. This document doesn't define them, but future documents might define them